tutorials/04 - DistortionEffectDemo.md

```diff
@@ -0,0 +1,138 @@
+# 04 - Distortion Effect
+
+In this tutorial, we're going to analyze the structure of an audio effect application, by applying distortion to a looping audio file.
+
+## Main Tracktion Engine Classes
+#### Engine
```

Comment:
As this is part 4 of a series. You probably don't need to repeat the classes that have already been covered.
What should probably be here though are `Plugin` and `PluginManager` as I don't think they're in previous tutorials.